### ISSUE TITLE
Ensure that quorum sets can be changed in flight

### ISSUE THREAD

**Issue by MonsieurNicolas** [stellar/stellar-core#876]
In the event of some major problem, we should make sure that nodes can be restarted with a different quorum set while proceeding in the current SCP round.

**Comment by jedmccaleb** [comment]
this is what this was about: #790

**Comment by MonsieurNicolas** [comment]
no, this is different: this one is about allowing a node to be restarted *during* an scp round and change its quorum set with the restart.
Right now, because of the way SCP works, it will not do anything and sit there as it waits to see a quorum based on its previous message: the SCP protocol does not order statements using quorum set, so we need to force bump the ballot in this situation to advertise to the network the new quorum set.

**Comment by jedmccaleb** [comment]
Ok so this needs to happen in addition to #790

**Comment by MonsieurNicolas** [comment]
so to do this, we'll have to move to the latest version of SCP that doesn't set b to infinite in the commit phase: that way we can just bump b's counter when we want to change the quorum set

**Comment by MonsieurNicolas** [comment]
Adding to 11.2 to start tracking changes needed for this
